<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnDesk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>WallPaper</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>561</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblToday">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Today Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTodayDate">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>2023/ 12 / 31</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSunrise">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Sunrise  :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRiseTime">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> 6:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSunset">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Sunset :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSetTime">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>17:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="moon">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="wallpaperButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>350</y>
          <width>211</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Set WallPaper</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listHour">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>34</y>
          <width>256</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblImg">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>241</width>
          <height>181</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lblCurTime">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>260</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Current Time: 00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Theme</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="serialtxt"/>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QComboBox" name="cmbTheme"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>serialNumber</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLabel" name="lblregister">
          <property name="text">
           <string>Registered To:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QComboBox" name="cmbwalls"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>reload app after setting wallpaper</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Wallpaper</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="0" colspan="2">
            <widget class="QCheckBox" name="hchimechk">
             <property name="text">
              <string>HourChime</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QComboBox" name="comboBox">
             <property name="maxVisibleItems">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>+1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+7 MST</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+6 MST DST</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QCheckBox" name="tockchk">
             <property name="text">
              <string>TickTock Noise</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Longitude</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Latitude</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Timezone</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="tztxt">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="longtxt">
             <property name="text">
              <string>110</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lattxt">
             <property name="text">
              <string>51</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="lblCountry">
               <property name="text">
                <string>Country  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbCountry">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblCity">
               <property name="text">
                <string>  City Name </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbCity">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="geobutton">
               <property name="text">
                <string>Find</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
